.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
Please include the `node_modules` folder in your `.gitignore` file.

<img width="846" alt="Screen Shot 2022-06-27 at 16 34 58" src="https://user-images.githubusercontent.com/82426682/176039434-93853ee6-5eab-4c4e-97ac-a77834bf5fc4.png">

Reply:
Hello, I didn't add the node_modules to the .git ignore file because I don't have any node_module file in my project. It wasn't a requirement, [these](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) were the instructions I got.